codecov.yml

```diff
@@ -18,3 +18,5 @@ coverage:
         if_ci_failed: error
         informational: true
         only_pulls: false
+github_checks:
+  annotations: false
```

Comment:
Should we remove the `comment: false` at the top?

Reply:
Right now, I like checking the codecov website to see coverage stats but don't really have a desire to use it for PRs.  So that's why I left it disabled at the top.  I think if we're ready to start demanding tests with PRs then it would make sense to set `comment: true`.  Do we feel ready for that?  Or do we want to go a little longer before then?  (I think maybe once dericcrago and dmsimard are onboard might be a good time to start demanding tests?)

Reply:
I think we should first add some basic tests for most of the basic functionality (like building docs and building ansible) before it makes sense to demand more tests.